tensorflow_serving/model_servers/main.cc

```diff
@@ -101,6 +101,17 @@ using tensorflow::serving::PredictionService;
 
 namespace {
 
+void ParseProtoTextFile(const string& file, google::protobuf::Message* message) {
```

Comment:
I've pushed with the bool version. If you still want Status just let me know (with a little guidance) and I'll fix it up.

Thank you!